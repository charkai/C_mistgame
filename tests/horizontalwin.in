place A1
place A12
place B1
place A13
place C1
place A14
place D1
place A15
place E1
