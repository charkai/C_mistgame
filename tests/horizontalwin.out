Black wins!
A1A12B1A13C1A14D1A15E1
Thank you for playing!
